src/libsyntax/feature_gate/builtin_attrs.rs

```diff
@@ -265,6 +265,9 @@ pub const BUILTIN_ATTRIBUTES: &[BuiltinAttribute] = &[
     // FIXME: #14408 whitelist docs since rustdoc looks at them
     ungated!(doc, Whitelisted, template!(List: "hidden|inline|...", NameValueStr: "string")),
 
+    // RFC #2008
+    ungated!(non_exhaustive, Whitelisted, template!(Word)),
```

Comment:
I would group this with the "Modules, ..." "section" that's probably where it will end up in the reference's attribute index.

Reply:
Fixed, thanks.